tauri/src/alchemy/global.rs

```diff
@@ -0,0 +1,27 @@
+use crate::{db::DB, types::GlobalState};
+use async_trait::async_trait;
+use once_cell::sync::OnceCell;
+use tokio::sync::{RwLock, RwLockReadGuard, RwLockWriteGuard};
+
+use super::Alchemy;
+
+static ALCHEMY: OnceCell<RwLock<Alchemy>> = OnceCell::new();
+
+#[async_trait]
+impl GlobalState for Alchemy {
+    type Initializer = DB;
+
+    async fn init(db: Self::Initializer) {
+        let instance = Alchemy::new(db);
+        instance.fetch_balances().await.unwrap();
```

Comment:
I don't think we should call this here (especially because it will receive arguments)

probably the real solution is that the UI itself will send an event asking for the balances to be refreshed (e.g.: if the GUI window isn't open, we don't even need to bother updating)